docs/reference/gnoclient/client.md

```diff
@@ -17,31 +17,39 @@ type Client struct {
 }
 ```
 
-### func \(\*Client\) [Call](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L56>)
+### func \(\*Client\) [AddPackage](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L236>)
+
+```go
+func (c *Client) AddPackage(cfg BaseTxCfg, msgs ...MsgAddPackage) (*ctypes.ResultBroadcastTxCommit, error)
+```
+
+`AddPackage` executes one or more [AddPackage](#type-msgaddpackage) calls on the blockchain.
+
+### func \(\*Client\) [Call](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L62>)
 
 ```go
 func (c *Client) Call(cfg BaseTxCfg, msgs ...MsgCall) (*ctypes.ResultBroadcastTxCommit, error)
 ```
 
 `Call` executes a one or more [MsgCall](#type-msgcall) calls on the blockchain.
 
-### func \(\*Client\) [Send](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L176>)
+### func \(\*Client\) [Send](<https://github.com/gnolang/gno/blob/master/gno.land/pkg/gnoclient/client_txs.go#L182>)
```

Comment:
Updating this every time will start to become really annoying and ultimately inaccurate over time. 
A better solution could be to redirect the user to the Go docs, which should serve as the reference for any GoLand public API, and maintain this page for links, references, examples, and usage. 
In the meantime, perhaps we should consider using a permalink to prevent inaccuracies in the documentation?

Reply:
Yeah, I need to reconsider how to do this the most optimal way. I'll keep it like this for now, as there is another effort coming up for gnoclient. Btw, I did not do these changes manually, but with a godoc to markdown tool.